inst/using-the-dccvalidator-app-amp-ad.Rmd

```diff
@@ -46,25 +52,26 @@ requirements, validating and reviewing the metadata, and uploading the dataset.
 
 ## General Process Overview
 
-1. Contact the AMP-AD team to discuss the study and the expected data. Receive
+1. Contact the Data Liaison (juliane.schneider@sagebase.org) to discuss the study and the expected data. Receive
 staging folder synIDs for each expected dataset.
-2. Upload documentation and validate metadata + manifest files in [dccvalidator](https://shinypro.synapse.org/users/kwoo/dccvalidator-app/).
-3. Contact the AMP-AD team when all files pass validation. The team will verify
+2. Using the templates located at the bottom of the [Onboarding page](https://www.synapse.org/#!Synapse:syn24986618), create a study description, methods description(s), and acknowledgement statement.
+3. Upload the study description, methods description and acknowledgement statement using the [Submission Form](https://www.synapse.org/#!Synapse:syn25051271)
+4. Validate metadata + manifest files in [dccvalidator](https://shinypro.synapse.org/users/kwoo/dccvalidator-app/).
+5. Contact the AMP-AD team when all files pass validation. The team will verify
 items not checked by the dccvalidator. Receive permissions to upload data to the
 staging folder.
-4. Use the validated manifest to upload the data with `syncToSynapse` (see
+6. Use the validated manifest to upload the data with `syncToSynapse` (see
 [Synapse documentation](https://docs.synapse.org/articles/uploading_in_bulk.html) for uploading data in bulk).
-5. Contact the AMP-AD team. The team will do the final verifications before
+7. Contact the AMP-AD team. The team will do the final verifications before
 releasing the data.
 
 ## Documentation Upload
 
-Each study in AMP-AD has accompanying [documentation in the portal](https://adknowledgeportal.synapse.org/#/Explore/Studies?Study=syn8391648).
-You can submit your documentation through the dccvalidator app on the
-Documentation page. There should be a study description for the whole study, and
-an assay description for each of the assays that was performed. These can be in
-a single file, or you can upload multiple files to the assay description
-section.
+Each study in AMP-AD has the accompanying [documentation in the portal](https://adknowledgeportal.synapse.org/#/Explore/Studies?Study=syn8391648):
```

Comment:
For each of these, study description, methods description, etc, should there be a short blurb about what they are for? This is only pointing them to templates without any context.

Reply:
@Aryllen I did that on purpose. I want them to have to go to the template without any assumptions/preconceptions, otherwise I think they'll tend to take that list and not pay much attention to the details of the template. What do you think?

Reply:
@pitviper6 I see your point! I'm fine with having it be bare-bones as in your original text. Same for these below.